docs/csharp/async.md

```diff
@@ -2,129 +2,128 @@
 
-2. Will your code be performing a very expensive computation?
+1. Will your code be performing an expensive computation?
 
-    If you answered "yes", then your work is **CPU-bound**.
+  If you answered "yes", then your work is **CPU-bound**.
 
-If the work you have is **I/O-bound**, use `async` and `await` *without* `Task.Run`.  You *should not* use the Task Parallel Library.  The reason for this is outlined in the [Async in Depth article](../standard/async-in-depth.md).
+If the work you have is **I/O-bound**, use `async` and `await` *without* `Task.Run`. You *should not* use the Task Parallel Library. The reason for this is outlined in the [Async in Depth article](../standard/async-in-depth.md).
 
-If the work you have is **CPU-bound** and you care about responsiveness, use `async` and `await` but spawn the work off on another thread *with* `Task.Run`.  If the work is appropriate for concurrency and parallelism, you should also consider using the [Task Parallel Library](../standard/parallel-programming/task-parallel-library-tpl.md).
+If the work you have is **CPU-bound** and you care about responsiveness, use `async` and `await` but spawn off the work on another thread *with* `Task.Run`. If the work is appropriate for concurrency and parallelism, you should also consider using the [Task Parallel Library](../standard/parallel-programming/task-parallel-library-tpl.md).
 
-Additionally, you should always measure the execution of your code.  For example, you may find yourself in a situation where your CPU-bound work is not costly enough compared with the overhead of context switches when multithreading.  Every choice has its tradeoff, and you should pick the correct tradeoff for your situation.
+Additionally, you should always measure the execution of your code. For example, you may find yourself in a situation where your CPU-bound work is not costly enough compared with the overhead of context switches when multithreading. Every choice has its tradeoff, and you should pick the correct tradeoff for your situation.
 
-## More Examples
+## More examples
 
-The following examples demonstrate various ways you can write async code in C#.  They cover a few different scenarios you may come across.
+The following examples demonstrate various ways you can write async code in C#. They cover a few different scenarios you may come across.
 
-### Extracting Data from a Network
+### Extracting data from a network
 
-This snippet downloads the HTML from the homepage at [www.dotnetfoundation.org](https://www.dotnetfoundation.org) and counts the number of times the string ".NET" occurs in the HTML.  It uses ASP.NET MVC to define a web controller method which performs this task, returning the number.
+This snippet downloads the HTML from the homepage at <https://dotnetfoundation.org> and counts the number of times the string ".NET" occurs in the HTML. It uses ASP.NET to define a Web API controller method, which performs this task, returning the number.
```

Comment:
```suggestion
This snippet downloads the HTML from the homepage at <https://dotnetfoundation.org> and counts the number of times the string ".NET" occurs in the HTML. It uses ASP.NET to define a Web API controller method, which performs this task and returns the number.
```

Reply:
I haven't seen angle brackets used for links like that. What is the purpose?

Reply:
It treats it as a link, simplifies it. They were wrong, since the `www.` isn't needed with this site and redirects anyways. So rather than updating it, I simplified it.